### PR TITLE
adding synthetics apm page

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
Adds a page about integrating synthetics with APM

### Motivation
PM request

### Preview link
https://docs-staging.datadoghq.com/cswatt/synthetics-with-apm/synthetics/apm/

### CODE REVIEW

File: content/en/synthetics/apm.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+<div class="alert alert-warning">Synthetics is in beta for the Datadog US Site. To request access, complete the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics request form</a>.</div>
+
+{{< img src="synthetics/apm/synthetics-apm.gif" alt="APM and Synthetics" responsive="true" style="width:80%;">}}
+
+## Overview
+
+The APM integration with Synthetics allows you to go from a test run that potentially failed to the root cause of the issue by looking at the trace generated by that very test run.
+
+Having both network related details thanks to your test and backend information thanks to your trace allows you to access a whole new level of details in the way your application is behaving as experienced by your user.
```
Comment (apigirl) [inline content/en/synthetics/apm.md]:
```suggestion
Having both network related specifics, thanks to your test, and backend information, thanks to your trace, allows you to access a whole new level of details in the way your application is behaving as experienced by your user.
```

File: content/en/synthetics/apm.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Datadog uses the distributed tracing protocol and sets up the following HTTP headers:
+
+* `x-datadog-trace-id`: Generated from the Synthetics backend. Allows Datadog to link the trace with the test result.
+* `x-datadog-parent-id`: 0
+* `x-datadog-origin`: Synthetics to make sure the generated traces don't affect your APM quotas. See below.
+* `x-datadog-sampling-priority`: 1, [to make sure that the Agent keeps the trace][2].
+
+### How are APM quotas affected?
+
+The `x-datadog-origin` header specifies to the APM backend that the traces were generated by Synthetics. The generated traces consequently do not impact your classical APM quotas.
```
Comment (apigirl) [inline content/en/synthetics/apm.md]:
```suggestion
The `x-datadog-origin` header specifies to the APM backend that the traces are Synthetics generated. The generated traces consequently do not impact your classical APM quotas.
```

File: content/en/synthetics/apm.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Create a check that hits your traced HTTP server, and Datadog automatically links the trace generated by your server to the corresponding check result.
+
+### How are traces linked to checks?
+
+Datadog uses the distributed tracing protocol and sets up the following HTTP headers:
+
+* `x-datadog-trace-id`: Generated from the Synthetics backend. Allows Datadog to link the trace with the test result.
+* `x-datadog-parent-id`: 0
+* `x-datadog-origin`: Synthetics to make sure the generated traces don't affect your APM quotas. See below.
+* `x-datadog-sampling-priority`: 1, [to make sure that the Agent keeps the trace][2].
```
Comment (MargotLepizzera) [inline content/en/synthetics/apm.md]: I believe it needs to be `x-datadog-sampling-priority: 1`

File: content/en/synthetics/apm.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+* Your HTTP server is using a library that supports distributed tracing.
+
+Create a check that hits your traced HTTP server, and Datadog automatically links the trace generated by your server to the corresponding check result.
+
+### How are traces linked to checks?
+
+Datadog uses the distributed tracing protocol and sets up the following HTTP headers:
+
+* `x-datadog-trace-id`: Generated from the Synthetics backend. Allows Datadog to link the trace with the test result.
+* `x-datadog-parent-id`: 0
```
Comment (MargotLepizzera) [inline content/en/synthetics/apm.md]: I believe it needs to be `x-datadog-parent-id: 0`

File: content/en/synthetics/apm.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+* `x-datadog-origin`: Synthetics to make sure the generated traces don't affect your APM quotas. See below.
```
Comment (MargotLepizzera) [inline content/en/synthetics/apm.md]: It needs to be `x-datadog-origin: synthetics`

File: content/en/synthetics/apm.md (hunk 1 above)
Comment (MargotLepizzera) [reply content/en/synthetics/apm.md]: From GJ: Having both network related specifics, thanks to your test, and backend, infra and log information, thanks to your trace, allows you to access a whole new level of details in the way your application is behaving as experienced by your user.

**Pull request merged**